corehq/apps/callcenter/views.py

```diff
@@ -65,11 +65,11 @@ def data_sources(self):
             (self.get_static_options_size, self.get_static_options),
             (self.get_groups_size, self.get_groups),
             (self.get_locations_size, self.get_locations),
-            (self.get_users_size, self.get_users),
+            (self.get_all_users_size, self.get_all_users),
```

Comment:
This view is used in the domain settings page here: https://github.com/dimagi/commcare-hq/blob/fe885ffedb943c7b559db2429a902299238d8b4a/corehq/apps/domain/forms.py#L555-L561
I'd think that place would want only active users, although I'm not positive.  At any rate, I'd think the safer bet would be to preserve the current behavior.